storage/cache/subtree_cache.go

```diff
@@ -170,3 +192,53 @@ func (s *SubtreeCache) Flush(setSubtree SetSubtreeFunc) error {
 	}
 	return nil
 }
+
+// makeSuffix creates a suffix key for indexing into the subtree's Leaves and
+// InternalNodes maps.
+func makeSuffix(depth int, index *big.Int) ([]byte, error) {
+	// TODO(al): only supports 8 bit subtree sizes currently
+	sfx := make([]byte, 2)
+	if depth >= 8 {
```

Comment:
could tidy up the error message to "... depths up to 8 ..." as it currently reads like it must be 8.